CRM/Utils/Token.php

```diff
@@ -1235,9 +1235,7 @@ public static function getTokenDetails(
       }
     }
 
-    $query = new CRM_Contact_BAO_Query($params, $returnProperties);
```

Comment:
Really - this is being substantiated & then called statically in a way that shows no benefit from the substantiation